CHANGELOG.md

```diff
@@ -3,6 +3,229 @@
 <!-- prettier-ignore-start -->
 
 
+## 0.8.0
+
+Since we released uv [0.7.0](https://github.com/astral-sh/uv/releases/tag/0.5.0) in April, we've
+accumulated various changes that improve correctness and user experience, but could break some
+workflows. This release contains those changes; many have been marked as breaking out of an
+abundance of caution. We expect most users to be able to upgrade without making changes.
+
+This release also includes the stabilization of a couple `uv python install` features, which have
+been available under preview since late last year.
+
+### Breaking changes
+
+- **Install Python executables into the bin ([#14626](https://github.com/astral-sh/uv/pull/14626))**
+
+  `uv python install` now installs a versioned Python executable (e.g., `python3.13`) into a
+  directory on the `PATH` (e.g., `~/.local/bin`) by default. This behavior has been available under
+  the `--preview` flag since [Oct 2024](https://github.com/astral-sh/uv/pull/8458). This change
+  should not be breaking unless it shadows a Python executable elsewhere on the `PATH`.
+
+  To install unversioned executables, e.g., `python3` and `python`, use the `--default` flag. The
+  `--default` flag has also been in preview, but is not stabilized in this release.
+
+  As with tool installation, the target directory respects common variables like `XDG_BIN_HOME` and
+  can be overridden with a `UV_PYTHON_BIN_DIR` variable.
+
+  You can opt out of this behavior with `uv python install --no-bin` or `UV_PYTHON_INSTALL_BIN=0`.
+
+  See the [documentation on installing Python
```

Comment:
VS Code also uses that to discover interpreters (https://learn.microsoft.com/en-us/visualstudio/python/managing-python-environments-in-visual-studio?view=vs-2022). I don't fully understand how else VS Code finds interpreters, so I don't know how much impact that has for VS Code users.